packages/slice-machine/components/ScreenshotChangesModal/index.tsx

```diff
@@ -0,0 +1,205 @@
+import Card from "@components/Card";
+import { AiOutlinePicture } from "react-icons/ai";
+import { RiErrorWarningLine } from "react-icons/ri";
+import SliceMachineModal from "@components/SliceMachineModal";
```

Comment:
no idea!

Reply:
do you want me to change it?

Reply:
Since there are already aliases, why change to relative path? Maybe it’s a more global topic aliases VS relative path that should be tackle in a different PR if you reach a consensus.